services/core-web/src/components/noticeOfWork/NOWProgressActions.js

```diff
@@ -105,7 +105,7 @@ export class NOWProgressActions extends Component {
   };
 
   handleStartDelay = (values) => {
-    this.props
+    return this.props
```

Comment:
I prefer implicit returns, removing the brackets would have the same effect I believe?

handleStartDelay = (values) => this.props....

Reply:
Agreed, sure I can do that